tensorflow/lite/kernels/internal/reference/resize_bilinear.h

```diff
@@ -212,9 +212,8 @@ inline void ResizeBilinearInteger(
               (input_y - (1 << 10) * y0) * (input_x - (1 << 10) * x0);
           const int64_t output_20 =
               output_20_ll + output_20_lu + output_20_rl + output_20_ru;
-          const int64_t round = (output_20 > 0) ? (1 << 19) : -(1 << 19);
-          const T interpolation =
-              static_cast<T>((output_20 + round) / (1 << 20));
+          const int64_t round = 1 << 19;
```

Comment:
How about using the macro to store two implementations, one for two rounding and one for one rounding mode?

Reply:
Effectively, now that https://github.com/tensorflow/tensorflow/pull/50290 is merged and the behaviour is behind a TFLITE_SINGLE_ROUNDING flag, I kept the old resize bilinear rounding behaviour and put the new single-rounding one behind TFLITE_SINGLE_ROUNDING.

Thanks